apps/src/lib/kits/maker/boards/microBit/MicroBitConstants.js

```diff
@@ -11,3 +12,13 @@ export const EXTERNAL_PINS = [0, 1, 2];
 export function roundToHundredth(rawValue) {
   return Math.floor(rawValue * 100) / 100;
 }
+export const MB_COMPONENT_COUNT = 8;
```

Comment:
why is this not `MB_COMPONENTS.length`?

Reply:
Good question. I added a comment to explain, but the short answer is that there are 6 unique component types and 8 components on the board (the components on the board include a component for the board itself, one of each type of components, except two of the Button type because there are two unique buttons on the board).